src/actions/countAction.js

```diff
@@ -0,0 +1,27 @@
+import { COUNT_UP } from '../actionTypes';
+import { COUNT_DOWN } from '../actionTypes';
+import { COUNT_RESET } from '../actionTypes';
+
+export function countUp() {
+  return dispatch => {
```

Comment:
no need to return function which takes `dispatch`.
this function should return action object